packages/appium/docs/en/ecosystem/build-drivers.md

```diff
@@ -390,11 +390,56 @@ You can add custom CLI args if you want your driver to receive data from the com
 Appium server is started (for example, ports that a server administrator should set that should not
 be passed in as capabilities.
 
-To do this you define a `schema` field within the `appium` field in `package.json`, pointing to
-a JS or JSON file defining an argument schema. (See the schema in `fake-driver`, for example).
-Arguments you define there will be available as a `--driver-<name>-<arg-name>` flag on the command
-line. If you define `my-arg` in the schema, for example, and if your driver's name is `awesome`,
-then users will be able to set your arg via `--driver-awesome-my-arg`.
+To define CLI arguments (or configuration properties), your extension must provide a _schema_. In
```

Comment:
```suggestion
To define CLI arguments (or configuration properties) for the Appium server, your extension must provide a _schema_. In
```

Might want to be specific that we're talking about server config, not e.g., script args.